anchore_engine/clients/docker_registry.py

```diff
@@ -216,9 +216,12 @@ def get_repo_tags(userId, image_info, registry_creds=None):
         url = "https://"+registry
         repo = image_info['repo']
 
-    lookuptag = 'latest'
-    if image_info['tag']:
-        lookuptag = image_info['tag']
+    if use_lookup_tag:
+        lookuptag = 'latest'
+        if image_info['tag']:
+            lookuptag = image_info['tag']
+    else:
+        lookuptag = None
```

Comment:
This doesn't work in current skopeo implementation if there is no 'latest' tag. See #484

Reply:
Ah, I tested it against one of my repos and it worked but the repo had a 'latest' tag. Is that what you mean?

Reply:
Yeah, a repo that doesn't have a 'latest' tag requires that the call include a tag to key off for skopeo. #484 removes that requirement but introduces some regression testing. However, I don't think we should add an API call with a parameter that will quickly be removed. We should probably update skopeo before this change goes in.

Reply:
Okay, I've assigned #484 to myself. Will work on that before merging the commits for this issue